src/platform/packages/shared/response-ops/rule_form/src/rule_details/rule_dashboards.tsx

```diff
@@ -38,7 +38,62 @@ export const RuleDashboards = ({ contentManagement }: Props) => {
     () => formData.artifacts?.dashboards ?? [],
     [formData.artifacts]
   );
+  const onChange = (selectedOptions: Array<EuiComboBoxOptionOption<string>>) => {
+    const artifacts = {
+      ...formData.artifacts,
+      dashboards: selectedOptions.map((selectedOption) => ({
+        id: selectedOption.value,
+      })),
+    };
+    dispatch({
+      type: 'setRuleProperty',
+      payload: {
+        property: 'artifacts',
+        value: artifacts,
+      },
+    });
+  };
 
+  return (
+    <>
+      <EuiSpacer size="l" />
+      <EuiFlexGroup>
+        <EuiFlexItem data-test-subj="ruleLinkedDashboards">
+          <EuiFormRow
+            label={
+              <LabelWithTooltip
+                labelContent={ALERT_LINK_DASHBOARDS_TITLE}
+                tooltipContent={ALERT_LINK_DASHBOARDS_LABEL_TOOLTIP_CONTENT}
+              />
+            }
+            fullWidth
+            labelAppend={OptionalFieldLabel}
+          >
+            <DashboardsCombobox
+              contentManagement={contentManagement}
+              dashboardsFormData={dashboardsFormData}
+              onChange={onChange}
+              placeholder={ALERT_LINK_DASHBOARDS_PLACEHOLDER}
+            />
+          </EuiFormRow>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </>
+  );
+};
+
+// TODO: Where should this component live?
+export const DashboardsCombobox = ({
```

Comment:
I extracted this component from the RuleDashboards component and I want it to reuse it in the form to create/edit SLOs.

Where should I move the DashboardsCombobox component?